code/__HELPERS/mobs.dm

```diff
@@ -578,6 +578,14 @@ GLOBAL_LIST_INIT(skin_tone_names, list(
 		if(mob.ckey == key)
 			return mob
 
+/// Returns a string for the specified body zone. If we have a bodypart in this zone, refers to its plaintext_zone instead.
+/mob/living/proc/parse_zone_with_bodypart(zone)
```

Comment:
im bad at naming procs. this might be ass. i do not know

Reply:
It's ok.